### DISCUSSION TITLE
中文 Logo 字体投票与征集

### DISCUSSION THREAD

**Discussion by greyli** [greyli/flask-docs-zh#32] [Vote]
下面是几个 Logo 备选项，欢迎投票选择你喜欢的版本（**点评论左边的向上箭头**）。

同时也欢迎提交你设计的版本（一条评论上传一张），只需要修改右下角的文字为中文即可（图片源文件在[这里](https://github.com/pallets/flask/raw/main/docs/_static/flask-logo.png)）。英文「Web/web」建议使用 Flask logo 的字体，即 High Tower Text。

相关 Issue：#26

**Comment by greyli** [comment]
<img alt="flask-logo-new" src="https://user-images.githubusercontent.com/12967000/119634909-c447e900-be45-11eb-88ee-341a7c1fb496.png">

High Tower Text + 思源简中（Noto Sans S Chinese Medium）

**Reply by RRRoger** [reply]
思源就挺好

**Comment by greyli** [comment]
![image](https://user-images.githubusercontent.com/12967000/119635151-f9543b80-be45-11eb-82aa-8790869271e1.png)

High Tower Text + 华文楷体

**Reply by zhengxiaowai** [reply]
华文楷体，有版权

**Reply by greyli** [reply]
> 华文楷体，有版权

@frostming 那这个选项就去掉啦？

**Reply by frostming** [reply]
那么 High Tower Text 如何，Windows中自带不等于能自由使用

**Reply by greyli** [reply]
从[这里](https://fonts.adobe.com/fonts/hightower#licensing-section)看，High Tower Text 可以用作 logo 字体和商用。

**Reply by greyli** [reply]
可以考虑替换为方正楷体。

**Comment by greyli** [comment]
![image](https://user-images.githubusercontent.com/12967000/119635212-08d38480-be46-11eb-938c-db3a927a8a69.png)

High Tower Text + 方正老报宋

**Reply by greyli** [reply]
@frostming 方正老报宋似乎也不能免费商用？或许可以替换为方正书宋。

**Reply by ian-quinn** [reply]
感觉衬线体之间比较搭，稍微粗点的宋体就好

**Comment by greyli** [comment]
![image](https://user-images.githubusercontent.com/12967000/119841927-cf2f7600-bf38-11eb-9114-7b296c21cb0a.png)

不作翻译。

**Comment by greyli** [comment]
几个可以免费商用的备选项：方正黑体、方正书宋、方正仿宋、方正楷体。有时间补一下效果图。

**Reply by ZhaoQi99** [reply]
Ubunt Mono考虑嘛？👀

**Reply by frostming** [reply]
英文字体没有争议，现在是确定中文字体

**Reply by 180909** [reply]
思源黑体考虑吗，是开源的。

**Reply by frostming** [reply]
第一个方案就是
